packages/block-editor/src/components/inner-blocks/style.scss

```diff
@@ -13,7 +13,7 @@
 }
 
 // On fullwide blocks, don't go beyond the canvas.
-[data-align="full"] > [data-block] .has-overlay::after {
+[data-align="full"] .has-overlay::after {
```

Comment:
I'd be cautious about this kind of changes because potentially there's children blocks, do we want to match the has-overlay for the fully aligned block or all its children.

Reply:
I think this selector will do the same. There's nothing between the outer and inner wrapper anymore. So this will still target all overlays within the block.